### PR TITLE
Example server to expose a fetch tool

### PR DESCRIPTION
**Pull request by dsp-ant**

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Python SDK Examples
+
+This folders aims to provide simple examples of using the Python SDK. Please refer to the
+[example servers repository](https://github.com/modelcontextprotocol/example-servers)
```
Comment (jspahrsummers) [inline examples/README.md]:
```suggestion
[example-servers repository](https://github.com/modelcontextprotocol/example-servers)
```

File: examples/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+for real-world servers.
+
+* **simple-tool**: provides a simple fetch tool
```
Comment (jspahrsummers) [inline examples/README.md]: Is `simple-prompt` missing from this list? Or should we just get rid of the list?

If we keep it, I think we should hyperlink to the actual folders.

File: examples/servers/simple-tool/mcp_simple_tool/__init__.py [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (jspahrsummers) [inline examples/servers/simple-tool/mcp_simple_tool/__init__.py:1]: Missing README for this server

File: examples/servers/simple-tool/mcp_simple_tool/server.py [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+import click
+import httpx
+import mcp.types as types
+from mcp.server import Server
+
+
+async def fetch_website(
+    url: str,
+) -> list[types.TextContent | types.ImageContent | types.EmbeddedResource]:
+    async with httpx.AsyncClient(follow_redirects=True) as client:
```
Comment (jspahrsummers) [inline examples/servers/simple-tool/mcp_simple_tool/server.py]: I think we should give this a dummy user-agent so requests are traceable to anyone using this code

File: examples/README.md (hunk 2 above)
Comment (dsp-ant) [reply examples/README.md]: I am just gonna get rid of it. I think I'll forget to keep it in sync.

File: examples/servers/simple-tool/mcp_simple_tool/__init__.py (hunk 3 above)
Comment (dsp-ant) [reply examples/servers/simple-tool/mcp_simple_tool/__init__.py:1]: good catch

**Pull request merged**